docs/static/scripts/xss/src/containers/Search.js

```diff
@@ -35,7 +35,7 @@ class Search extends Component {
       .catch(() => this.setState({ error: true }))
   }
 
-  clearInput = () => {
+  clearInputFunc = () => {
```

Comment:
Pourquoi?

Reply:
I was getting an error that was ambiguous and hard to track down at first. Part of it was making me think that because the first input tag's id was also `clearinput` that React was being weird and confusing these two. It obviously def wasn't that, but I just didn't change it back.